driver-core/src/test/java/com/datastax/driver/core/CCMBridge.java

```diff
@@ -1089,7 +1097,16 @@ public void run() {
     }
 
     private static boolean isThriftSupported(VersionNumber cassandraVersion) {
-      return cassandraVersion.compareTo(VersionNumber.parse("4.0")) < 0;
+      // Thrift is removed from some pre-release 4.x versions, make the comparison work for those
+      return cassandraVersion.nextStable().compareTo(VersionNumber.parse("4.0")) < 0;
+    }
+
+    private static boolean isMaterializedViewsSupported(VersionNumber cassandraVersion) {
+      return cassandraVersion.nextStable().compareTo(VersionNumber.parse("4.0")) >= 0;
```

Comment:
MVs are available for C* 3+, but indeed they are disabled by default in C* 4+.

If the intent of this method is to detect whether or not we need to include `enable_materialized_views` in C* options, then its name is misleading. I suggest `isMaterializedViewsDisabledByDefault`.

See JAVA-2183.

Reply:
Good point, this makes more sense.